examples/onboard-aws-redshift/README.md

```diff
@@ -0,0 +1,66 @@
+# Onboard Amazon Redshift example
+This example includes additional prerequisites that will need to be completed to fully utilize the module. More details can be found in the [onboarding documentation](https://docs.imperva.com/bundle/onboarding-databases-to-sonar-reference-guide/page/Amazon-Redshift-Onboarding-Steps_48367115.html).
+
+It creates both 'aws' and 'dsfhub' resources. More information regarding authentication to each one can be found in the relevant provider documentation:
+- [aws](https://registry.terraform.io/providers/hashicorp/aws/latest/docs)
+- [dsfhub](https://registry.terraform.io/providers/imperva/dsfhub/latest/docs)
+
+Importantly, the example includes two different audit methods: auditing via the Audit Tables, or auditing via an S3 bucket. <b>Please choose your desired audit method</b>.
+
+## Option 1: Auditing via Audit Tables
+## Prerequisites
```

Comment:
I fix these in my PR in the private repo, but this and `Database Configuration` should be a `###` and same thing with the subheadings under Option 2